deepmd/nvnmd/utils/encode.py

```diff
@@ -0,0 +1,172 @@
+
+import numpy as np
+import logging 
+
+from deepmd.nvnmd.data.data import jdata_sys
+
+log = logging.getLogger(__name__)
+
+
+class Encode():
```

Comment:
A lot of codes don't have the unit test, for example this class. See details of converge:

https://app.codecov.io/gh/deepmodeling/deepmd-kit/compare/1707/tree/deepmd/nvnmd